### PR TITLE
Integrate client information card with API

### PR DESCRIPTION
**Pull request by abelayt**

### CODE REVIEW

File: app/controllers/api/clients_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  class ClientsController < ActionController::API
+    def client_by_id
+      clients = Clients::ClientRepository.new.client_by_id params[:client_id]
```
Comment (edgibbs) [inline app/controllers/api/clients_controller.rb]: So this should probably just be a default restful route for Rails which would make this show and would default to having an :id passed in as a param.

File: app/controllers/clientid_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ClientidController < ApplicationController
+    def index; end
+  end
```
Comment (edgibbs) [inline app/controllers/clientid_controller.rb]: newline issue

File: config/routes.rb [hunk 3]
```diff
@@ -16,5 +16,11 @@
         get ':user_id', to: 'referrals#referrals_by_user'
       end
     end
+    resources :clients, only: [ :show ] do
+      collection do
+
```
Comment (edgibbs) [inline config/routes.rb]: switch to just a restful route

javascript
```
resources :clients, only [:show]
```

Comment (edgibbs) [reply config/routes.rb]: You'll just need to rename the client_by_id() to show() and you'll get an id param instead of client_id

**Comment by edgibbs** [comment]
Still working out the details with the state Jenkins servers, but the build is failing which you can see here: http://jenkins.dev.cwds.io:8080/job/cm-pull-request/

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

File: docker-compose.yml [hunk 4]
```diff
@@ -15,7 +15,7 @@ services:
     ports:
       - "6379:6379"
   case-api:
-    image: ${REPOSITORY}/case-management-api
+    image: ${REPOSITORY}/case-management-api:no-auth-tmp
```
Comment (eckdanny-osi) [inline docker-compose.yml]: just noticed this in `master` at 78e0d9e after tending to PRs today... why was the default tag (`latest`) changed to `no-auth-tmp`?

**Pull request merged**